tools/tests_wycheproof_generate_ecdh.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Random "Randy" Lattice and Sean Andersen
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
+'''
+Generate a C file with ECDH testvectors from the Wycheproof project.
+'''
+
+import json
+import sys
+
+from binascii import hexlify, unhexlify
+from wycheproof_utils import to_c_array
+
+def should_skip_flags(test_vector_flags):
+    # skip these vectors because they are for ASN.1 encoding issues and other curves
+    flags_to_skip = {"InvalidAsn", "WrongCurve"}
```

Comment:
A short description list of all the (unclear) categories would be nice. This has confused us, and I assume it will confuse others in the future. (Or just add another reference to https://github.com/bitcoin-core/secp256k1/pull/1492#discussion_r1572491546 )